pkg/types/exchange.go

```diff
@@ -153,8 +153,9 @@ type ExchangeRewardService interface {
 }
 
 type TradeQueryOptions struct {
-	StartTime   *time.Time
-	EndTime     *time.Time
-	Limit       int64
-	LastTradeID uint64
+	StartTime      *time.Time
+	EndTime        *time.Time
+	Limit          int64
+	LastTradeID    uint64
+	MinimumTradeID uint64
```

Comment:
you can add a comment for this MinimuTradeID? thanks

Reply:
ok